Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/HalfPeriodRatio.mo

```diff
@@ -0,0 +1,180 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay;
+block HalfPeriodRatio
+  "Calculates the half period ratio of a response from a relay controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput tOn
+    "Connector for a signal of the length for the On period"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+    iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput tOff
+    "Connector for a signal of the length for the Off period"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+    iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput rho
+    "Connector for a real signal of the half period ratio"
+    annotation (Placement(transformation(extent={{100,50},{120,70}}),
+        iconTransformation(extent={{100,50},{120,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput triggerStart
+    "Connector for a boolean signal, true if the tuning starts"
+    annotation (Placement(
+        transformation(extent={{100,-10},{120,10}}), iconTransformation(extent={
+            {100,-10},{120,10}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput triggerEnd
+    "Connector for a boolean signal, true if the tuning completes"
+    annotation (Placement(
+        transformation(extent={{100,-70},{120,-50}}), iconTransformation(extent=
+           {{100,-70},{120,-50}})));
+  Buildings.Controls.OBC.CDL.Continuous.Min tmin
+    "The minimum value of tOn and tOff"
+    annotation (Placement(transformation(extent={{-80,20},{-60,40}})));
+  Buildings.Controls.OBC.CDL.Continuous.Greater gretOntOff
+    "Check if tOn*tOff*min(tOn*tOff) is larger than 0"
```

Comment:
avoid use any parameter names in the comments. If in the future the parameter is changed to different name, then we cannot understand the comment.

Reply:
Fixed it and other similar comments as well